docs-app/public/docs/3-demos/external-column-ordering.md

```diff
@@ -27,7 +27,11 @@ export default class extends Component {
 
   changeColumnOrder = () => {
     this.pendingColumnOrder = new ColumnOrder({
-      columns: () => this.columns,
+      allColumns: () => this.columns,
+      availableColumns: () => this.columns.reduce(function(acc, col) {
```

Comment:
sorry it's taken me so long to get to this PR. <3


can you add a comment in this demo's code (maybe inline) about why `ColumnOrder` takes two separate collections of the columns?